.travis.yml

```diff
@@ -25,7 +25,7 @@ script:
     - ./run_checkdocs.py
     - ./run_unittests.sh
     - ./run_pylint.sh --FF
-    - inspekt style --disable E501,E265,W601,E402,E731
+    - inspekt style --disable E501,E265,W601,E402,E731,E221
```

Comment:
(I think you're reviewing the whole shebang, not commit-by-commit. That will make it impossible to follow the thread).

Because pep8 is all-or-nothing: there's no way to disable a warning in just one place using comments.